src/xenia/kernel/xam/content_manager.cc

```diff
@@ -243,8 +243,7 @@ X_RESULT ContentManager::DeleteContent(const XCONTENT_DATA& data) {
   auto global_lock = global_critical_region_.Acquire();
 
   auto package_path = ResolvePackagePath(data);
-  if (xe::filesystem::PathExists(package_path)) {
-    xe::filesystem::DeleteFolder(package_path);
+  if (std::filesystem::remove_all(package_path) > 0) {
     return X_ERROR_SUCCESS;
   } else {
     return X_ERROR_FILE_NOT_FOUND;
```

Comment:
This could be replaced with the following:
```c++
  if (std::filesystem::remove_all(package_path) > 0) {
    return X_ERROR_SUCCESS;
  } else {
    return X_ERROR_FILE_NOT_FOUND;
  }
```
From the documentation, `std::filesystem::remove_all`:
> Returns the number of files and directories that were deleted (which may be zero if p did not exist to begin with).

This would cover the case of the directory not existing, but there are corner cases of the OS not removing these paths for whatever reason, in which case:
> The overload that does not take a `std::error_code&` parameter throws filesystem_error on underlying OS API errors, constructed with p as the first path argument and the OS error code as the error code argument.

We may not want any exceptions thrown here, in which case, maybe we should use the version which uses `std::error_code&` and interpret the OS API error code and return that.

On the other hand, these would be issues already present in Xenia without this change.